src/Translator.java

```diff
@@ -2,14 +2,20 @@
 import java.util.HashMap;
 
 public class Translator {
-    Map<Integer,String>numericAlpha = new HashMap<>();
 
-//    for(int i = 0; i < numeric.length; i++) {
-//
-//    };
-//
-//     for(int i = 0; i < alfabetic.length; i++) {
-//
-    };
+    static Map<Integer,String> numericAlpha = new HashMap<>();
 
-}
+
+   public Translator(String[] alphabetic, int[] numeric){
```

Comment:
Je hebt overal eerst de int en dan de string, probeer die constantheid vast te houden in je code dus:

`public Translator(int[] numeric, String[] alphabetic){`